templates/index.html

```diff
@@ -27,8 +27,9 @@ <h4>Getting Started</h4>
         If you aren't yet familiar with GraphQL, see the GraphQL Resources section below. Otherwise, start poking around and find loopholes! There are GraphQL
         Implementation flaws as well as general application vulnerabilities.
       </p>
-      <p>If you are interacting with DVGA programmatically, you can set a specific game mode (such as Beginner, or Expert) by passing the HTTP Request Header <code>X-DVGA-MODE</code> with either <code>Beginner</code> or <code>Expert</code> as values.</p>
-      <p>If the Header is not set, DVGA will default to <u>Easy mode</u>.</p>
+      <p>You can set a "game mode" in DVGA: A beginner level or expert level by clicking on the top bar menu's cube icon and choosing the level. This is a global setting that will apply to all clients (GUI or CLI)</p>
+      <p>If you are interacting with DVGA programmatically, you can also set the game mode by passing the HTTP Request Header <code>X-DVGA-MODE</code> set to either <code>Beginner</code> or <code>Expert</code> as values.</p>
+      <p>If the Header is not set, DVGA will default to <u>Easy mode</u> or to whatever you previously set in the user interface.</p>
```

Comment:
Instead of "Easy mode" call it Beginner Mode

Reply:
grave-level mistake